core/src/main/resources/error/error-classes.json

```
@@ -1603,6 +1603,11 @@
     ],
     "sqlState" : "42000"
   },
+  "WRITE_STREAM_NOT_ALLOWED" : {
+    "message" : [
+      "`writeStream` can be called only on streaming Dataset/DataFrame"
```

Add a dot at the end please:
```suggestion
      "`writeStream` can be called only on streaming Dataset/DataFrame."
```